src/utils/web3converter.py

```diff
@@ -0,0 +1,42 @@
+from src.typings import SlotNumber, EpochNumber, FrameNumber
+from src.modules.submodules.typings import ChainConfig, FrameConfig
+
+
+class Web3Converter:
+    """
```

Comment:
Add link to spec
https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_epoch_at_slot